Makefile

```diff
@@ -204,9 +204,9 @@ distclean: clean
 
 setup:
 	$(MAKE) -C tools
+	mkdir -p baserom
```

Comment:
Is this line necessary? There's a line with a comment saying something like 
```
# create build directories
```
And it has `baserom` on it.

Reply:
That creates `build/*`  This creates `baserom/`  I need to do it here because the C program failed if the folder doesn't exist.

Reply:
What about adding `baserom` to that line?

Reply:
I could but it seems that its there for making `build`. I'll add it and see if anyone objects.